src/search/parser/abstract_syntax_tree.cc

```diff
@@ -404,7 +404,7 @@ const plugins::Type &ListNode::get_type(DecorateContext &context) const {
 }
 
 LiteralNode::LiteralNode(Token value)
```

Comment:
To make this more consistent with other places, I'd pass in a const ref here and copy the value in the initialization list, rather than passing in a copy and then moving the copy.